lib/vjs-hls.js

```diff
@@ -30,7 +31,49 @@ var attachVideojsStreamrootProvider = function (window, videojs, Hls) {
         }
 
         function _onError(event, data) {
-            console.error('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details);
+            var error = {
+              message: ('HLS.js error: ' + data.type + ' - fatal: ' + data.fatal + ' - ' + data.details),
+            };
+            console.error(error.message);
+
+            // increment/set error count
+            errorCounts[data.type] ? errorCounts[data.type] += 1 : errorCounts[data.type] = 1;
```

Comment:
That's weird I don't have this problem. I've checked out your branch and I'm able to access `_errorCounts` or `tech` correctly from the `StreamrootProviderHLS` scope without additional arguments.
Are you sure it's not a build / watch problem on your side?